contrib/devtools/symbol-check.py

```diff
@@ -135,24 +132,30 @@
 
 PE_ALLOWED_LIBRARIES = {
 'ADVAPI32.dll', # security & registry
+'CRYPT32.dll', # crypto functions
 'IPHLPAPI.DLL', # IP helper API
 'KERNEL32.dll', # win32 base APIs
 'msvcrt.dll', # C standard library for MSVC
 'SHELL32.dll', # shell API
 'USER32.dll', # user interface
 'WS2_32.dll', # sockets
 # bitcoin-qt only
+'COMDLG32.DLL', # common dialogs like "open file"
+'comdlg32.dll', # same as above but lowercase in win64 binary
 'dwmapi.dll', # desktop window manager
 'GDI32.dll', # graphics device interface
-'IMM32.dll', # input method editor
+'IMM32.DLL', # input method editor
+'IMM32.dll', # same as above but lowercase extension in win64 binary
```

Comment:
I'm not sure I understand why the extension is lowercase in one example but the entire filename is lowercase in another. There's another example later.

Do we need a consistency path, or a case normalization mapping somewhere? Or do we handle this on an exceptional basis as we see them?

Reply:
It's weird yeah I noticed the same thing a while back and brought that up but my understanding is that we can follow up with optimizations like uniform casing on the next round of updates.

Reply:
Yeah I was thinking of drafting something up in a follow up PR. I would think adding a function to normalize the binary.name on the check would be the most flexible so long as the name matches but that's just my initial thought. Still unclear why there's a discrepancy in the first place.

Reply:
I ended up moving to `i686-pc-linux-gnu` last night and got stuck on that per lack of interface overrides which is to be expected. Let me start a remote build with 64-bit mingw to confirm.